com.unity.render-pipelines.high-definition/Runtime/Lighting/ScreenSpaceLighting/HDRenderPipeline.ScreenSpaceGlobalIllumination.cs

```diff
@@ -52,42 +52,61 @@ IndirectDiffuseMode GetIndirectDiffuseMode(HDCamera hdCamera)
             return mode;
         }
 
+        int CombineIndirectDiffuseHistoryStateToMask(bool fullResolution, bool rayTraced)
```

Comment:
I assume all stuff in this file is because this branch is based on something not in master yet?

Reply:
Oh no I see you changed the bools to a mask here